src/status_im/contexts/wallet/common/sheets/account_options/view.cljs

```diff
@@ -32,12 +32,13 @@
 
 (defn- options
   [{:keys [theme show-account-selector? options-height]}]
-  (let [{:keys [name color emoji address watch-only?]} (rf/sub [:wallet/current-viewing-account])
-        network-preference-details                     (rf/sub [:wallet/network-preference-details])
-        multichain-address                             (utils/get-multichain-address
-                                                        network-preference-details
-                                                        address)
-        share-title                                    (str name " " (i18n/label :t/address))]
+  (let [{:keys            [name color emoji address watch-only?]
+         default-account? :wallet} (rf/sub [:wallet/current-viewing-account])
```

Comment:
Extracting the `:wallet` key to check if it's the default account or not seems error-prone. We might change the structure of the db and we won't notice this is breaking.

cc: @J-Son89 is exracting `:wallet` the proper way to check if it's the default account? is the`:created-at` key reliable to check this?

@Rende11 
I think the proper way to achieve it is adding a `:default-account?` key when we receive the accounts to store.

The event is `:wallet/get-accounts-success` in the ns `status-im.contexts.wallet.events`, it'd be really good if you check which is the default account and add the previously mentioned keyword, it'll make the info clearer and also availabe to all the subs

Reply:
It is correct to check :wallet key.
We probably should rename it when we store it in the db to :default-account?

Reply:
I added `:default-account?` key - see src/status_im/contexts/wallet/data_store.cljs